src/services/signatureHelp.ts

```diff
@@ -525,32 +534,35 @@ function getSpreadElementCount(node: SpreadElement, checker: TypeChecker) {
     return 0;
 }
 
-function getArgumentCount(argumentsList: Node, ignoreTrailingComma: boolean, checker: TypeChecker) {
+function getArgumentCount(argumentsList: Node, checker: TypeChecker) {
     // The argument count for a list is normally the number of non-comma children it has.
     // For example, if you have "Foo(a,b)" then there will be three children of the arg
     // list 'a' '<comma>' 'b'.  So, in this case the arg count will be 2.  However, there
     // is a small subtlety.  If you have "Foo(a,)", then the child list will just have
     // 'a' '<comma>'.  So, in the case where the last child is a comma, we increase the
     // arg count by one to compensate.
-    //
-    // Note: this subtlety only applies to the last comma.  If you had "Foo(a,," then
-    // we'll have: 'a' '<comma>' '<missing>'

```

Comment:
The mention of `<missing>` was added here like 9 years ago but the missing node is not used here for years already (I checked against some 3.x versions). When dealing with `fn(,,,,)` we just get a list of comma tokens